src/main/java/org/jboss/snowdrop/licenses/xml/LicenseElement.java

```diff
@@ -29,26 +31,33 @@
 
     private String url;
 
+    private String textUrl;
```

Comment:
Why is this needed? Because only `textUrl` or `url` ever is used there is no reason to have them both in here. Set appropriate value to `url` when creating an object.